recipe/meta.yaml

```diff
@@ -19,10 +19,13 @@ requirements:
     - pip
     - python >=3.6
     - setuptools
+    - scipy ==1.7.3
+    - numpy =1.22
   run:
     - python >=3.6
     - numba >=0.45
-    - scipy >=0.16,<1.6.2
+    - scipy >=0.16,<=1.7.3
+    - numpy <1.23
```

Comment:
This should be fixed on the `scipy` feedstock. Can you report an issue there?

Reply:
sure, but it's and old version of `scipy` doubt there is much interest in fixing that.

Reply:
> This should be fixed on the scipy feedstock. Can you report an issue there?

doesn't look like they keep 1.7.* branch around:

https://github.com/conda-forge/scipy-feedstock/branches

anyway, feel free to close this PR

Reply:
https://github.com/conda-forge/scipy-feedstock/issues/216